### DISCUSSION TITLE
Remove all implants, beacons and/or profiles at once

### DISCUSSION THREAD

**Discussion by Krishna8009** [BishopFox/sliver#1000] [General]
Similiar to how we have Jobs -K to kill all listeners , is there a way i can delete all the implants i have created at once , something like - implants rm all 
Presently , the way to remove implant that i know is implant rm <implant_name>. This becomes tedious if we have a lot of implants created. 
Similiarly , for beacons we can prune all inactive beacons, but how can i kill and prune all beacons together ? 

I am new to using sliver c2, so i may not know all commands . but i have reviewed the help options many times and cant seem to find a way

**Comment by moloch--** [comment]
There's not really a "nuke everything" option in the UI as this functionality typically isn't desirable. However, you can reset everything by removing the `SLIVER_ROOT_DIR` which by default will be `~/.sliver` (as the server). For example, `rm -rf ~/.sliver`.

**Reply by Krishna8009** [reply]
Thankyou for sharing :)
